javascript/builtins/intl/DateTimeFormat.json

```diff
@@ -354,63 +212,370 @@
+                    },
+                    "ie": {
+                      "version_added": false
+                    },
+                    "nodejs": {
+                      "version_added": "12.0.0"
+                    },
+                    "opera": {
+                      "version_added": "60"
+                    },
+                    "opera_android": {
+                      "version_added": "52"
+                    },
+                    "safari": {
+                      "version_added": "13"
+                    },
+                    "safari_ios": {
+                      "version_added": "13"
+                    },
+                    "samsunginternet_android": {
+                      "version_added": "11.0"
+                    },
+                    "webview_android": {
+                      "version_added": "73"
+                    }
+                  },
+                  "status": {
+                    "experimental": false,
+                    "standard_track": true,
+                    "deprecated": false
+                  }
+                }
+              },
+              "options_timeStyle_parameter": {
+                "__compat": {
+                  "description": "<code>options_timeStyle</code> parameter",
+                  "support": {
+                    "chrome": {
+                      "version_added": "76"
+                    },
+                    "chrome_android": {
+                      "version_added": "76"
+                    },
+                    "edge": {
+                      "version_added": "79"
+                    },
+                    "firefox": {
+                      "version_added": "79"
+                    },
+                    "firefox_android": {
+                      "version_added": "79"
+                    },
+                    "ie": {
+                      "version_added": false
+                    },
+                    "nodejs": {
+                      "version_added": "12.9.0"
+                    },
+                    "opera": {
+                      "version_added": "63"
+                    },
+                    "opera_android": {
+                      "version_added": "54"
+                    },
+                    "safari": {
+                      "version_added": "14.1"
+                    },
+                    "safari_ios": {
+                      "version_added": "14.5"
+                    },
+                    "samsunginternet_android": {
+                      "version_added": "12.0"
+                    },
+                    "webview_android": {
+                      "version_added": "76"
+                    }
+                  },
+                  "status": {
+                    "experimental": false,
+                    "standard_track": true,
+                    "deprecated": false
+                  }
+                }
+              },
+              "options_timeZone_parameter": {
```

Comment:
So this creates a timeZone option and adds the iana stuff as a subfeature - both with same compat. I think this is probably wrong - timezone came first. Looking at versions, perhaps timeZone should be added as earliest version for the few cases where it is not already the earliest version?

Reply:
> Looking at versions, perhaps timeZone should be added as earliest version for the few cases where it is not already the earliest version?

Yeah, this seems more plausible to me, too

Reply:
thanks!